tools/prepare_lava_job.py

```diff
@@ -0,0 +1,169 @@
+)
+
+parser.add_argument(
+    "-l",
+    "--lava",
+    dest="lava",
+    type=str,
+    nargs='?',
+    const="True",
+    help="submit generated job with lavacli",
+)
+
+def run_lavacli() -> bool:
+    try:
+        pkg_resources.get_distribution("lavacli")
+    except pkg_resources.DistributionNotFound:
+        print("lavacli is not installed...")
+        return False
+    
+    lavacli_cmd = LAVACLI_JOB_SUBMIT + NEW_JOB_NAME
+    process = subprocess.Popen(lavacli_cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = process.communicate()
+    stdout = stdout.decode('utf-8')
+    stderr = stderr.decode('utf-8')
+    
+    print(f"{lavacli_cmd}")
+    print(stdout)
+
+    if stderr:
+        print("lavacli Error:")
+        print(stderr)
+        return False
+
+    return True
+
+#TODO what about another URLs???
+def get_imagename_from_url(url):
+    return os.path.basename(url)
+
+def add_namespace_in_testsuite(content):
+    actions = content.get("actions", [])
+
+    for action in actions:
+        if "test" in action or "boot" in action or "deploy" in action:
+            action_name = next(iter(action))
+            action[action_name]["namespace"] = "testsuite"
+
+    return content
+
+def create_job(device_type, image_url, file_flash, file_testsuite) -> bool:
+    print("Generate Job... ", end="")
+
+    # Load the contents of file_flash
+    try:
+        with open(file_flash, 'r') as file:
+            file_flash_contents = yaml.safe_load(file)
+    except OSError:
+        print(f"\nCould not read file to flash device: {file_flash}")
+        return False
+    print(file_testsuite)
+    # Load the contents of file_testsuite
+    try:
+        with open(file_testsuite, 'r') as file:
+            file_testsuite_contents = yaml.safe_load(file)
+    except OSError:
+        print(f"\nCould not read file with testsuite: {file_testsuite}")
+        return False
+    print("blablabla... ")
+    if device_type:
+        # Update device_type
+        file_flash_contents['device_type'] = device_type
+    
+    if not image_url:
+        image_url = file_flash_contents['actions'][2]['deploy']['images']['recovery_image']['url']
+
+    # Update job_name
+    file_flash_contents['job_name'] = 'Flash & Test Image: '+ get_imagename_from_url(image_url)
+    
+    if image_url:
+        # Update the image URL with the provided image parameter
+        file_flash_contents['actions'][2]['deploy']['images']['recovery_image']['url'] = image_url
+
+    # Append the actions from "file_testsuite" to the actions in "file_flash"
+    file_flash_contents['actions'].extend(file_testsuite_contents['actions'])
+    
+    # Add or modify namespace from "file_testsuite"
+    file_testsuite_contents = add_namespace_in_testsuite(file_testsuite_contents)
+
+    # Write the updated contents to NEW_JOB_NAME with proper indentation
+    try:
+        with open(NEW_JOB_NAME, 'w') as file:
```

Comment:
We should dump the temporary job to submit to a tempdir and clean it up again, i.e. https://docs.python.org/3/library/tempfile.html or `$TMPDIR`.